packages/lib-react-components/src/PlainButton/PlainButton.js

```diff
@@ -13,18 +13,15 @@ export const StyledPlainButton = styled(Button)`
 `
 
 function PlainButton (props) {
-  const { className, onClick, labelSize, text } = props
+  const { className, onClick, labelSize, text, textColor } = props
```

Comment:
Is it possible to use `color` as the prop name here? That more closely aligns with the Grommet [terminology](https://v2.grommet.io/text#color), and I think it would make sense intuitively since many of our components closely mirror Grommet terms.

Reply:
Renamed prop from ‘textColor’ to ‘color’. Just by renaming this prop type to match Grommet’s api, it fixed the underline color to match the text color!